### PR TITLE
raw http to unauth server

### PR DESCRIPTION
**Pull request by MHBauer**
no more shelling out to kubectl

### CODE REVIEW

**Comment by MHBauer** [comment]
@duglin why is travis blowing up?

**Comment by duglin** [comment]
I have no idea.  Its building w/o docker for some reason - its had golang installed but we didn't ask for it.  But even so, I can't reproduce this locally.

**Comment by duglin** [comment]
I think travis is messing up by making golang available, but if that's it then we probably need to set GOPATH.
Try adding:
```
GOAPTH=$(shell cd ../../.. && pwd)
```
to the root Makefile right before the "all:" target and see if that helps.

**Comment by MHBauer** [comment]
@duglin travis documentation is awful, but it does say the environments come with go installed by default. No idea what's going on. Tried a couple things. It's green, but it doesn't look quite right to me. I don't think it built the new client?

I'll let you fix it next week.

**Comment by MHBauer** [comment]
@duglin https://www.traviscistatus.com/incidents/11hp8bhkrkn7 might be why stuff started breaking. They made changes.Added go 1.6.3 by default.

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,6 @@ services:
   - docker
 
 script:
+  - go version
+  - go get -d ./...
```
Comment (duglin) [inline .travis.yml]: you can delete these lines,they're not needed any more.

**Comment by MHBauer** [comment]
@duglin okay. it worked.

**Comment by MHBauer** [comment]
lgtm myself

**Pull request merged**